kafka-impl/src/test/java/io/streamnative/pulsar/handlers/kop/format/EntryFormatterTest.java

```diff
@@ -104,22 +120,25 @@ public void testEntryFormatterEncode(CompressionType compressionType, byte magic
 
         MemoryRecords records = prepareRecords(compressionType, magic);
 
+        PartitionLog.LogAppendInfo appendInfo =
```

Comment:
@BewareMyPower See this class.

Reply:
I see. That's because you pass the `LogAppendInfo` to the constructor of `EncodeRequest`.

Generally, a class with default access modifier should only be used in its package. But since you want classes in other packages to access this class (`LogAppendInfo`), then it should be exposed as public class.

Reply:
So in this PR, making `LogAppendInfo` a static inner class is okay.